docs/tracing/index.md

```diff
@@ -56,28 +56,28 @@ TIME(s)        UID    COMM             PID     TID     RET             EVENT
 ```
 
 !!! Note
-    There are 2 ways to enable tracing only:  
-    1. To export a TRACEE_EBPF_ONLY=1 env variable to docker.  
-    2. To provide a `trace` 1st argument to docker container.  
+    There are 2 ways to enable tracing only:
+    1. To export a TRACEE_EBPF_ONLY=1 env variable to docker.
+    2. To provide a `trace` 1st argument to docker container.
 
 Each line is a single event collected by Tracee-eBPF, with the following
 information:
 
-1. **TIME**  
```

Comment:
This is tricky, those 2 spaces at the end are formatting for mkdocs (you will have to run makedocs locally and make sure they are all good, I took special care of that last time I edited this).

```
make -f builder/Makefile.mkdocs
```